payloads/external/skiboot/Makefile

```diff
@@ -0,0 +1,36 @@
+## SPDX-License-Identifier: GPL-2.0-only
+
+skiboot_dir=$(CURDIR)/talos-skiboot
+skiboot_git_repo=https://git.raptorcs.com/git/talos-skiboot
+skiboot_revision=9858186353f2203fe477f316964e03609d12fd1d
+dts=talos_ii_device-tree_stripped.dts
```

Comment:
We can't put files specific to one platform here. Probably another Kconfig option will be needed.

Reply:
I have added a Kconfig option `CONFIG_SKIBOOT_REVISION` and `CONFIG_SKIBOOT_DEVICE_TREE`